### PR TITLE
Updating Github actions to match the latest:

### PR DESCRIPTION
**Pull request by nsabri1**
History of update:
https://github.com/alphagov/smart-answers/commit/bf83acfea948d1c38506202d7b65a25999a507f6

⚠️ This repo is Continuously Deployed: make sure you [follow the guidance](https://docs.publishing.service.gov.uk/manual/development-pipeline.html#merge-your-own-pull-request) ⚠️

### CODE REVIEW

**Review by sengi — APPROVED** [review]
Thanks!

File: .github/workflows/deploy.yaml [hunk 1]
```diff
@@ -17,19 +17,18 @@ on:
...
-  push:
-    branches:
-      - main
-    paths-ignore:
-      - "Jenkinsfile"
-      - ".git**"
+  workflow_run:
+    workflows: [CI]
+    types: [completed]
+    branches: [main]
```
Comment (theseanything) [inline .github/workflows/deploy.yaml:23]: You might want to keep this as on push - until we've got CI set up properly

File: .github/workflows/set-automatic-deploys-enabled.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: Set automatic_deploys_enabled (optionally image_tag too)
+name: Set automatic_deploys
```
Comment (theseanything) [inline .github/workflows/set-automatic-deploys-enabled.yaml:1]: Nit: make it human friendly to read
 
```suggestion
name: Set automatic deploys
```

Comment (nsabri1) [reply .github/workflows/set-automatic-deploys-enabled.yaml:1]: Oh yes forgot to do that

File: .github/workflows/deploy.yaml (hunk 1 above)
Comment (nsabri1) [reply .github/workflows/deploy.yaml:23]: ok

**Review by theseanything — APPROVED** [review]

**Pull request merged**